ReactCommon/fabric/components/image/ImageShadowNode.cpp

```diff
@@ -48,7 +48,9 @@ ImageSource ImageShadowNode::getImageSource() const {
   auto sources = getProps()->sources;
 
   if (sources.size() == 0) {
-    return {.type = ImageSource::Type::Invalid};
+    return {
+        ImageSource::Type::Invalid // type
```

Comment:
As a note, most of the other Windows-compatible code in the fb monorepo that has migrated from designated initializers has done it with leading comments:

```
return {
    /* .type = */ ImageSource::Type::Invalid
};
```

That also makes it easier to codemod out the comment and return them to actual designated initializers when MSVC gets around to actually supporting them. (IIRC they have made it into C++20, so it shouldn't be too long a wait)